calcom/cal.com
# Hotfix - Add email field to profile settings

## What does this PR do?



This PR adds the email field to the profile settings

![image](https://user-images.githubusercontent.com/65426560/190454405-281366cb-912c-4827-8a45-3f9bc4b50be8.png)


Fixes # (issue)



**Environment**: Staging(main branch) / Production

## Type of change





## How should this be tested?



- Go to settings/my-account/profile
- Change your password and save changes
- In the DB or if you revisit the page the email should display the changed email

## Checklist



- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
